src/content/docs/new-relic-solutions/get-started/intro-new-relic2.mdx

```diff
@@ -0,0 +1,515 @@
+  - Get started
+translate:
+  - jp
+  - kr
+metaDescription: 'What is New Relic? No matter what is your business or role, learn how to make the most of our suite of digital intelligence products.'
+redirects:
+  - /docs/accounts-partnerships/education/getting-started-new-relic/introduction-new-relic
+  - /docs/accounts-partnerships/getting-started-new-relic/introduction-new-relic
+  - /docs/using-new-relic/welcome-new-relic/getting-started/introduction-new-relic
+  - /taxonomy/term/5766
+  - /docs/using-new-relic/welcome-new-relic/getting-started/what-new-relic-can-do-you
+  - /docs/new-relic-one/use-new-relic-one/core-concepts/new-relic-one-transition-guide-july-2020
+  - /docs/using-new-relic/welcome-new-relic/get-started/aW50cm9kdW
+  - /docs/using-new-relic/welcome-new-relic/get-started/introduction-new-relic
+  - /docs/using-new-relic/
+  - /docs/using-new-relic/welcome-new-relic
+  - /docs/using-new-relic/welcome-new-relic/get-started/intro-new-relic
+  - /docs.newrelic.com/docs/new-relic-solutions/new-relic-one/ui-data/new-relic-feature-end-life-announcements-july-2020
+  - /docs/new-relic-solutions/new-relic-one/install-configure/install-new-relic
+  - /docs/new-relic-solutions/get-started/quick-launch-guide
+  - /docs/using-new-relic/welcome-new-relic/get-started/get-started-full-stack-observability
+  - /docs/agents/manage-apm-agents/installation/install-agent
+  - /docs/new-relic-one/use-new-relic-one/cross-product-functions/install-configure/install-new-relic
+  - /docs
+  - /docs/using-new-relic/cross-product-functions/install-configure/install-new-relic
+signupBanner:
+  text: Monitor and improve your entire stack. 100GB free. Forever.
+freshnessValidatedDate: 2024-03-dd
+---
+
+import NRAlertsMain from 'images/alerts_screenshot-crop_main.webp'
+
+import NRApmMain from 'images/apm_screenshot-crop_main.webp'
+
+import NRBrowserMain from 'images/browser_screenshot-crop_main.webp'
+
+import NRDashboardsMain from 'images/dashboards_screenshot-crop_main.webp'
+
+import NRErrosInboxMain from 'images/errors-inbox_screenshot-crop_main.webp'
+
+import NRIastMain from 'images/iast_screenshot-crop_main.webp'
+
+import NRInfraMain from 'images/infrastructure_screenshot-crop_main.webp'
+
+import NRLambdaMain from 'images/lambda_screenshot-crop_main.webp'
+
+import NRLiveArchivesMain from 'images/live-archives_screenshot-crop_main.webp'
+
+import NRLogsMain from 'images/logs_screenshot-crop_main.webp'
+
+import NRK8sMain from 'images/kubernetes_screenshot-crop_main.webp'
+
+import NRMobileMain from 'images/mobile_screenshot-crop_main.webp'
+
+import NRNetworkMain from 'images/network_screenshot-crop_main.webp'
+
+import NROpenTelemetryMain from 'images/opentelemetry_screenshot-crop_main.webp'
+
+import NRServiceLevelsMain from 'images/slm_screenshot-crop_main.webp'
+
+import NRSyntheticMain from 'images/synthetic_screenshot-crop_main.webp'
+
+import NRVulnManagementMain from 'images/vulnerability-management_screenshot-crop_main.webp'
+
+
```

Comment:
I like the concept of how the screenshots are presented, I just think it may need to be more a bit more pronounced as on dark mode, the screenshots look a little rotated instead of tilted.